integration/terraform/ec2/linux/variables.tf

```diff
@@ -1,98 +1,85 @@
+variable "region" {
+  type    = string
+  default = "us-west-2"
+}
+
 variable "ec2_instance_type" {
-  type = string
+  type    = string
   default = "t3a.xlarge"
 }
 
 variable "key_name" {
-  type = string
+  type    = string
   default = "cwagent-integ-test-key"
 }
 
-variable "iam_instance_profile" {
-  type = string
-  default = "CloudWatchAgentServerRole"
-}
-
-variable "vpc_security_group_ids" {
-  type = list(string)
-  default = ["sg-013585129c1f92bf0"]
-}
-
-variable "region" {
-  type = string
-  default = "us-west-2"
-}
-
 variable "ami" {
-  type = string
+  type    = string
   default = "cloudwatch-agent-integration-test-ubuntu*"
 }
 
 variable "ssh_key" {
-  type = string
+  type    = string
   default = ""
 }
 
 variable "user" {
-  type = string
-  default = ""
-}
-
-variable "github_sha" {
-  type = string
-  default = ""
-}
-
-variable "github_repo" {
-  type = string
+  type    = string
   default = ""
 }
 
 variable "install_agent" {
   description = "command of package to install ex dpkg -i -E ./amazon-cloudwatch-agent.deb"
-  type = string
-  default = ""
+  type        = string
+  default     = ""
 }
 
 variable "ca_cert_path" {
-  type = string
+  type    = string
   default = ""
 }
 
 variable "arc" {
-  type = string
+  type    = string
   default = ""
 }
 
 variable "binary_name" {
-  type = string
+  type    = string
   default = ""
 }
 
 variable "local_stack_host_name" {
-  type = string
+  type    = string
   default = "localhost.localstack.cloud"
 }
 
 variable "s3_bucket" {
-  type = string
+  type    = string
   default = ""
 }
 
 variable "test_name" {
-  type = string
+  type    = string
   default = ""
 }
 
 variable "test_dir" {
-  type = string
+  type    = string
   default = ""
 }
-variable "sha" {
-  type = string
+
+variable "github_sha" {
+  type    = string
   default = ""
 }
-variable "sha_date"{
+
+variable "github_repo" {
+  type    = string
+  default = ""
+}
+
+variable "github_sha_date"{
```

Comment:
What are these changes for

Reply:
Changing from "sha_date" to "github_sha_date" to grouping the "Github variables" and also delete the `sha` which is a shorter version of "github_sha" when considering the impacts as can be found in this comment https://github.com/aws/amazon-cloudwatch-agent/pull/500#discussion_r915235575